astropy/io/votable/tree.py

```diff
@@ -3509,6 +3510,32 @@ def iter_groups(self):
     def iter_info(self):
         yield from self.infos
 
+    def pop(self, index=-1):
+        """
+        Remove and return item at index (default last).
+
+        Raises IndexError if list is empty or index is out of range.
```

Comment:
numpydoc convention allows "Raises" section. Also `index` would also need to be documented. Please see https://numpydoc.readthedocs.io/en/latest/format.html